### DISCUSSION TITLE
How to choose the order of Compose UI additionnal buttons ?

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [InboxSDK/InboxSDK#811] [Q&A]
<img width="373" alt="Capture d’écran 2022-12-07 à 09 25 34" src="https://user-images.githubusercontent.com/32040951/206126855-07e912e7-d921-4fe4-90e5-da7436ff7a0a.png">

I want to move the first button added with inbox sdk to the right.

Does someone know how can I achieve that ?

**Comment by wegry** [comment]
The SDK doesn't currently provide a way to position ComposeButtons to the right. See https://inboxsdk.github.io/inboxsdk-docs/compose/#composebuttondescriptor for more info.
